src/Views/Shared/ds-Header.cshtml

```diff
@@ -10,7 +10,7 @@
             <div class="govuk-header__content">
             @if (displayHeading)
             {
-                <a href="#" class="govuk-header__link govuk-header__link--service-name">
+                <a href="../@Model.BaseURL" class="govuk-header__link govuk-header__link--service-name">
```

Comment:
BaseURL is not within SuccessViewModel which breaks success pages for payment and non-payment journeys.